packages/wonder-blocks-modal/docs.md

```diff
@@ -54,3 +54,67 @@ const styles = StyleSheet.create({
     />
 </View>;
 ```
+
```

Comment:
Hmm, rather than adding all of these to a single docs.md file it'd be better to add this to a `modal-launcher.md` file (that way it's shown with the modal-launcher docs). Same goes for the two-column modal.

Reply:
Hmm nod, the reason I was avoiding this was because snapshot tests didn't pick them up, but maybe that's a quick fix we should just do.

Reply:
We need to update `gen-snapshot-tests.js` to handle that case.